cmd/humioctl/init_windows.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"os"
+
+	"golang.org/x/sys/windows"
+)
+
+// init in init_windows.go ensures that PowerShell and CMD emulate colors and handle
+// other escape codes properly.
+func init() {
+	stdout := windows.Handle(os.Stdout.Fd())
+	var originalMode uint32
+
+	windows.GetConsoleMode(stdout, &originalMode)
+	windows.SetConsoleMode(stdout, originalMode|windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING)
```

Comment:
maybe:

```suggestion
	if err := windows.GetConsoleMode(stdout, &originalMode); err == nil {
	    _ = windows.SetConsoleMode(stdout, originalMode|windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING)
	}
```

Reply:
Think that is overkill in this case.

Reply:
Wouldn't hurt, doesn't cost anything, and the change is already there?